src/views/SignUp/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Button } from '../../components';
+import {
+  extractUserData,
+  getUserFromRedirect,
+  handleSignIn,
+  saveUserInFirestore,
+} from './utils';
+import queryString from 'query-string';
+
+function SignUpBeforeRedirect() {
+  return (
+    <>
+    <Button onClick={() => handleSignIn()}>Sign Up Using Facebook</Button>
+    </>
```

Comment:
Can we put this in the header? instead of it being a separate page

Reply:
Maybe, but this would be a different product, I think that @guytepper doesn't want to encaurage anybody to sign-up, only people that want to claim ownership (be the leader) of a protest.